tic1.js

```diff
@@ -0,0 +1,127 @@
+    '\n '+cells[3]+' | '+cells[4]+' | '+cells[5]+'\n'+divider+
+    '\n '+cells[6]+' | '+cells[7]+' | '+cells[8]))
+    }
+
+let askForInputX = async function(){
+    let input = await ask('>');
+    if (cells[parseInt(input)-1] == 'X' || cells[parseInt(input)-1] == 'O'){
+        console.log("\nSorry! That space has been claimed! Try again.\n");
+        askForInputX();
+    }else{
+        for (let item of cells){
+            if (item === parseInt(input)){
+                cells[parseInt(input)-1] = 'X'
+            }
+        }
+        winCheck(cells);
+        playerO();
+}
+}
+
+let askForInputO = async function(){
+    let input = await ask('>');
+    if (cells[parseInt(input)-1] == 'X' || cells[parseInt(input)-1] == 'O'){
+        console.log("\nSorry! That space has been claimed! Try again.\n");
+        askForInputO();
+    }else{
+        for (let item of cells){
+            if (item === parseInt(input)){
+                cells[parseInt(input)-1] = 'O'
+            }
+        }
+        winCheck(cells);
+        playerX();
+    }
+}
+
+let winCheck = function(array){
+    let counter = 0;
+    if (array[0]==array[1] && array[1]==array[2]){
+        printGrid();
+        console.log('HOORAY! YOU WIN!')
+        process.exit();
+    }else if (array[3]==array[4] && array[4]==array[5]){
+        printGrid();
+        console.log('HOORAY! YOU WIN!')
+        process.exit();
+    }else if (array[6]==array[7] && array[7]==array[8]){
+        printGrid();
+        console.log('HOORAY! YOU WIN!')
+        process.exit();
+    }else if (array[0]==array[4] && array[4]==array[8]){
+        printGrid();
+        console.log('HOORAY! YOU WIN!')
+        process.exit(); 
+    }else if (array[2]==array[4] && array[4]==array[6]){
+        printGrid();
+        console.log('HOORAY! YOU WIN!')
+        process.exit(); 
+    }else if (array[0]==array[3] && array[3]==array[6]){
+        printGrid();
+        console.log('HOORAY! YOU WIN!')
+        process.exit();
+    }else if (array[1]==array[4] && array[4]==array[7]){
+        printGrid();
+        console.log('HOORAY! YOU WIN!')
+        process.exit();  
+    }else if (array[2]==array[5] && array[5]==array[8]){
+        printGrid();
+        console.log('HOORAY! YOU WIN!')
+        process.exit(); 
+}   else {for (let item of array){
+    if (item == 'X' || item == 'O'){
+        counter += 1;
+    }
+    if (counter === 9){
+        printGrid();
+        console.log("\nCAT'S GAME! Better luck next time!")
+        process.exit();
+    }
+}
+}
+
+}
+
+
+let playerX = async function(){
+    printGrid();
+    console.log("\nIt's your turn player 'X'! Please enter the number of the cell you'd like to claim!\n");
+    askForInputX();
+}
+
+
+let playerO = async function(){
+    printGrid();
+    console.log("\nIt's your turn player 'O'! Please enter the number of the cell you'd like to claim!\n");
+    askForInputO();
+}
+
+playerX();

```

Comment:
following up on above suggestion, instead of immediately calling `playerX`, call a new function `playGame` that switches between players and calls down to `playerX` and `playerO`, instead of the current deep nesting:

playerX -> askForInputX -> playerO -> askForInputO -> playerX -> ...